test/unit/org/apache/cassandra/utils/OverlapsTest.java

```diff
@@ -254,15 +254,14 @@ public void testAssignOverlapsIntoBuckets()
         };
         String[] none3 = new String[]{
         "ABCD",
-        "ADE",
-        "NPQ",
         "RST",
+        "NPQ",
         };
-        int[] noneUnselected = new int[]{2, 3, 4, 5};
+        int[] noneUnselected = new int[]{1, 5, 4, 2, 3};
         String[] single3 = new String[]{
         "ABCDE",
-        "LNOPQ",
         "RST",
+        "LNOPQ",
```

Comment:
Nit: As we don't care about the order here, let's save us some future pain by sorting the lists in `verifyAssignment` below.